lib/pymedphys/docs/release-notes.md

```diff
@@ -8,6 +8,87 @@ This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.35.0]
+
+### News around this release
+
+* Installation on MacOS (Intel) has been simplified and is now the same as for other platforms, thanks to work by the new pymssql maintainers.
+
+### "Stable" API changes
+
+(Won't truly be stable until after a 1.0.0 release)
+
+#### Breaking changes
+
+* DicomListener removed from public API (access is only through CLI)
```

Comment:
Pretty sure this API never got exposed within a release.

Reply:
You're right.  It was removed prior to 0.34.0
I'll remove that.